src/main/java/util/GithubPageElements.java

```diff
@@ -16,6 +16,10 @@
     public static final String COMMITS_TAB = "commits";
     public static final String FILES_TAB = "files";
 
+    public enum PrTab {
+        DISCUSSION, COMMITS, FILES
+    }
```

Comment:
Sorry, I just opened your branch. No need to change this enum, but remove the three unused variables above since you replaced them with the enum.